app/views/suppliers/orders/_delivery_method.html.erb

```diff
@@ -0,0 +1,18 @@
+<% if @shipping_address %>
+  <h5>Shipping Address:</h5>
+    <p>
+      <%= @shipping_address.line_one %></br>
```

Comment:
I've seen the 'break' tag written with the forward slash after the letter 'r' and I've seen it without a forward slash, but never in front.  Does it work the same way as a regular break when the slash is in the front?